ibis/expr/format.py

```diff
@@ -113,7 +113,7 @@ def get_result(self):
         elif isinstance(what, ops.TableColumn):
             text = self._format_column(self.expr)
         elif isinstance(what, ops.Literal):
-            text = 'Literal[{}]\n  {}'.format(
+            text = 'Literal[{}] {}'.format(
                 self._get_type_display(), str(what.value)
             )
```

Comment:
```suggestion
            text = 'Literal[{self._get_type_display()}] {what.value}'
```

Fine to keep as is if this puts you over the maximum line length